File: P21_u3(while).java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p21_u3;
import java.util.Scanner;
/**
 *
 * @author richy
 */
public class P21_u3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        int c=1, t, num, cub; // t= total num= numero cub= cubo
        System.out.println("Ingrese la cantidad de numeros: ");
        t = sc.nextInt();
        
    while(c<=t)
       {
        System.out.println("Ingrese su numero: ");   
        num = sc.nextInt();
        if(num>0)
        {
            cub = num * num * num;
            System.out.println("El resultado de la operacion es: " + cub);
        }
        c++;
       }
    }
}

File: p21_u3(while).cs
using System;

namespace p21_u3
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int c = 1, t, num, cub;
            Console.WriteLine("Ingrese la cantidad de numeros: ");
            t = Convert.ToInt32(Console.ReadLine());

            while (c <= t)
            {
                Console.WriteLine("Ingrese su numero: ");
                num = Convert.ToInt32(Console.ReadLine());
                if (num > 0)
                {
                    cub = num * num * num;
                    Console.WriteLine("El resultado de la operacion es: " + cub);
                }
                c++;
            }
        }
    }
}